### PR TITLE
[Android] Normalizing the dialogs about the Crosswalk Project runtime

### PR DESCRIPTION
**Pull request by lincsoon**
Improve the message and button label to make the dialogs more clearly

BUG=XWALK-4220

### CODE REVIEW

**Comment by rakuco** [comment]
@bspencer and others: please take a look at the strings here or in the ticket and provide some feedback.

**Comment by lincsoon** [comment]
Please reference to the JIRA issue that is easier to read
https://crosswalk-project.org/jira/browse/XWALK-4220

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 1]
```diff
@@ -2,62 +2,59 @@
...
       <message desc="Download Progress Message [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_PROGRESS_MESSAGE">
-        Downloading Crosswalk
-      </message>
-      <message desc="Download Failed Title [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_TITLE">
-        Crosswalk Download Failed
+        Downloading Crosswalk Project from remote host...
       </message>
       <message desc="Download Failed Message [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_MESSAGE">
-        Please retry or cancel
+        Download failed. An network error occurred.
```
Comment (halton) [inline runtime/android/core/strings/xwalk_app_strings.grd]: typo: `An network` -> `A network`

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 2]
```diff
@@ -2,62 +2,59 @@
...
-        Crosswalk Signature Verification Failed
+        Crosswalk Project Signature Verification Failed
       </message>
       <message desc="Startup Signature Check Error Message [CHAR-LIMIT=100]" name="IDS_STARTUP_SIGNATURE_CHECK_ERROR_MESSAGE">
-        The integrity of Crosswalk runtime library can not be verified. Please uninstall the runtime library and restart this application
+        The integrity of Crosswalk runtime library can not be verified.
       </message>
       <message desc="Startup Older Version Title [CHAR-LIMIT=100]" name="IDS_STARTUP_OLDER_VERSION_TITLE">
-        Crosswalk Version Too Old
+        New Crosswalk Project Runtime Required
```
Comment (halton) [inline runtime/android/core/strings/xwalk_app_strings.grd]: I think required should be lowercase. Anywaym let's ask @bspencer to decide.

Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: This is the title. So we should uppercase the first letter of all words

Comment (halton) [inline runtime/android/core/strings/xwalk_app_strings.grd]: That make sense, then you have LGTM. Need we wait for @bspencer's review?

Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: Yes, we need the feedback from @bspencer

**Comment by baleboy** [comment]
LGTM with nits (but would like @bspencer to approve)

"Decompressing Crosswalk Project..." change to "Decompressing Crosswalk Project runtime..."

"The integrity of Crosswalk Project can not be verified" change to "The integrity of Crosswalk Project cannot be verified"

**Comment by lincsoon** [comment]
Fixed nits above

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 3]
```diff
@@ -2,62 +2,59 @@
...
       <message desc="Download Failed Time Out [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_TIME_OUT">
-        Timed out
-      </message>
-      <message desc="Market Open Failed Title [CHAR-LIMIT=100]" name="IDS_MARKET_OPEN_FAILED_TITLE">
-        Market Open Failed
+        Download failed. The network connection timed out.
       </message>
       <message desc="Market Open Failed Message [CHAR-LIMIT=100]" name="IDS_MARKET_OPEN_FAILED_MESSAGE">
-        Please set up your market
+        Failed to open the app store
```
Comment (bspencer) [inline runtime/android/core/strings/xwalk_app_strings.grd]: Decide if you want "." at the end of errors and be consistent on all.  Also, for this error, is there anything the user can do?  The error doesn't help them solve it.   Something like:  "Failed to open the app store.  Please setup your app store account."  (just a guess)

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 4]
```diff
@@ -2,62 +2,59 @@
...
+        Mismatch of CPU Architecture for Crosswalk Project
       </message>
       <message desc="Startup Architecture Mismatched Message [CHAR-LIMIT=100]" name="IDS_STARTUP_ARCHITECTURE_MISMATCH_MESSAGE">
-        Please get Crosswalk Runtime Library
+        The Crosswalk Project must be updated to match the CPU Architecture of the device. Select "Continue" to download it.
       </message>
       <message desc="Startup Signature Check Error Title [CHAR-LIMIT=100]" name="IDS_STARTUP_SIGNATURE_CHECK_ERROR_TITLE">
-        Crosswalk Signature Verification Failed
+        Crosswalk Project Signature Verification Failed
       </message>
```
Comment (bspencer) [inline runtime/android/core/strings/xwalk_app_strings.grd]: Is "Signature Verification" a program?  Why is it capitalized?  What about:  "The Crosswalk Project signature verification failed."   Also, is there something they can do, like re-download it?  If so maybe add "Please re-install the Crosswalk Project runtime."  (just a guess)

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 5]
```diff
@@ -2,62 +2,59 @@
...
       <message desc="Startup Signature Check Error Message [CHAR-LIMIT=100]" name="IDS_STARTUP_SIGNATURE_CHECK_ERROR_MESSAGE">
-        The integrity of Crosswalk runtime library can not be verified. Please uninstall the runtime library and restart this application
+        The integrity of Crosswalk runtime library cannot be verified.
       </message>
```
Comment (bspencer) [inline runtime/android/core/strings/xwalk_app_strings.grd]: Why remove "Please uninstall..."  ?   Is it due to space limitations?  It can be helpful to guide them to fix the issue.  Also, I think we should add "the":  "The integrity of the Crosswalk Project runtime library cannot be verified."

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 6]
```diff
@@ -2,62 +2,59 @@
...
       <message desc="Startup Older Version Title [CHAR-LIMIT=100]" name="IDS_STARTUP_OLDER_VERSION_TITLE">
-        Crosswalk Version Too Old
+        New Crosswalk Project Runtime Required
       </message>
```
Comment (bspencer) [inline runtime/android/core/strings/xwalk_app_strings.grd]: A new Crosswalk Project runtime is required.   Or, if you want to be brief:  "New Crosswalk Project runtime required."

File: runtime/android/core/strings/xwalk_app_strings.grd [hunk 7]
```diff
@@ -2,62 +2,59 @@
...
       <message desc="Startup Older Version Message [CHAR-LIMIT=100]" name="IDS_STARTUP_OLDER_VERSION_MESSAGE">
-        Please update Crosswalk or continue
+        The version of Crosswalk Project on this system must be updated. Select "Continue" to download it.
       </message>
       <message desc="Startup Newer Version Title [CHAR-LIMIT=100]" name="IDS_STARTUP_NEWER_VERSION_TITLE">
-        Crosswalk Version Too New
+        Crosswalk Project Version Incompatible
       </message>
```
Comment (bspencer) [inline runtime/android/core/strings/xwalk_app_strings.grd]: "The Crosswalk Project version is incompatible."   or  "Incompatible Crosswalk Project version."

File: runtime/android/core/strings/xwalk_app_strings.grd (hunk 3 above)
Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: Actually, this is not related to the account. This error generally means there is no app store installed on the device. The end-user should install one, but we don't know what kind of app store is available for the end-user (Google Play Service is banned in some contries).

File: runtime/android/core/strings/xwalk_app_strings.grd (hunk 4 above)
Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: This is the title, so it's capitalized. We can put the instruction in the message bellow (The integraty of Crosswalk ...).

File: runtime/android/core/strings/xwalk_app_strings.grd (hunk 6 above)
Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: This is the title too. I think the "." is not necessary. Can I keep current one?

File: runtime/android/core/strings/xwalk_app_strings.grd (hunk 5 above)
Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: I will use "The integrity of the Crosswalk Project cannot be verified. Please uninstall the Crosswalk Project and restart the application.".

File: runtime/android/core/strings/xwalk_app_strings.grd (hunk 7 above)
Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: This is the title too. I will use "Incompatible Crosswalk Project Version".

File: runtime/android/core/strings/xwalk_app_strings.grd (hunk 3 above)
Comment (lincsoon) [inline runtime/android/core/strings/xwalk_app_strings.grd]: Can I use "Failed to open the app store. You need an app store to download the Crosswalk Project runtime.".

**Comment by lincsoon** [comment]
@bspencer I added missed "the" and more instructions to deal with the error. Please help to review again.

**Comment by baleboy** [comment]
The capitalisation needs to be consistent. Let's go with the style of using capital letters only for Crosswalk Project, so for example "New Crosswalk Project Runtime Required" should be "New Crosswalk Project runtime required". There are several instances like that, especially "Runtime".

**Comment by lincsoon** [comment]
@baleboy I understand your intention. But for the title, every word should be capitalized, isn't it? For this case:
Title: `New Crosswalk Project Runtime Required`
Message:`The version of the Crosswalk Project runtime on this system must be updated. Select "Continue" to download it.`

The message obeys the rule of "runtime".

**Comment by baleboy** [comment]
Ah, of course, I didn't realize those were titles. LGTM then, you can merge. We can always tweak later if needed.
